### ISSUE TITLE
Preinstalled cheat database 

### ISSUE THREAD

**Issue by lwcorp** [0ldsk00l/nestopia#334]
Just for comparison, Snes9X included preinstalled cheats  [back in 2018](https://github.com/snes9xgit/snes9x/releases/tag/1.56). All they did was take something existing like [Mighymo's](https://web.archive.org/web/20150831080933/http://www.mightymo.net:80/) **Nestopia Cheat Code Pack (2015-04-02)** (although hopefully [something newer](https://gamehacking.org/system/nes)) and include it inside the program.

In the slight chance the developerd change their mind and release a new version, I hope you'll consider this.

**Comment by Cibomatto2002** [comment]
The Latest version for windows 1.51

https://www.dosgamers.com/nintendo-nes/nestopia-nes-emulator

**Comment by lwcorp** [comment]
> The Latest version for windows 1.51

Yes, a new version was indeed released (actually even 1.51.1)...but unfortunately completely ignored this issue.
If new versions come out and continue the spirit of innovation, I hope they'll consider implementing it.

**Comment by rdanbrook** [comment]
This issue is not being "completely ignored". I can't simply distribute someone's cheat database without permission, and it seems that Mightymo is unreachable. I am open to shipping a cheat database, but everything needs to be above board.

**Comment by lwcorp** [comment]
Thanks for responding! Mightymo's cheats are ancient anyway (his site was closed in 2015).

But as suggested in my original post, why not contacting the new database? [https://gamehacking.org](https://gamehacking.org)'s opening paragraph is "Join our Discord to inquire how you can be involved, **or how your project can benefit from our site**."
